docs/guides/troubleshooting.md

```diff
@@ -15,6 +15,20 @@ title: Troubleshooting
   [this issue](https://github.com/facebook/WebDriverAgent/issues/507) for more details.
 * `shake` is implemented via AppleScript and works only on Simulator due to lack of support from Apple
 
+## Interact with dialogs managed by `com.apple.springboard`
+
+System dialogs, such as permission dialogs, might not be interactable directly when the active application is not `com.apple.springboard`
+because of the current active application. For example, when a session starts with `appium:app` or `appium:bundleId` capability with
```

Comment:
> because of the current active application

might be removed

Reply:
For example, when a session starts with  ...

This sentence is not really needed, the previous one already describes the same